messages/previewMessages.md

```diff
@@ -0,0 +1,39 @@
+# conflicts.none
+
+No conflicts found.
+
+# conflicts.header
+
+Conflicts [%s] Run the command with --ignore-conflicts to override
+
+# ignored.none
+
+No files are ignored because of .forceignore
```

Comment:
Hmm, not sure what this means.  So here are a few possible choices, based on my guess of what the issue is; if I didn't understand the issue right, can you update what you have similarly but correctly? Thx. 

"No files were ignored.  Check your .forceignore file for details on what you're currently ignoring."

or

"No files were ignored due to the current contents of your .forceignore file. "

or

"No files were ignored. Update your .forceignore file if you want to ignore certain files."

Reply:
I took the last one.